event/handler_test.go

```diff
@@ -1,29 +1,23 @@
 package event_test
 
 import (
-	"os"
 	"testing"
 
-	"github.com/ONSdigital/dp-search-reindex-tracker/config"
-	"github.com/ONSdigital/dp-search-reindex-tracker/event"
 	. "github.com/smartystreets/goconvey/convey"
 )
 
-// TODO: remove hello called example test
-func TestHelloCalledHandler_Handle(t *testing.T) {
+func TestReindexRequestedHandler_Handle(t *testing.T) {
+	Convey("Given a successful event handler, when Handle is triggered", t, func() {
+	})
+
+	Convey("handler returns an error", t, func() {
+	})
+}
 
+func TestSearchDataImportHandler_Handle(t *testing.T) {
```

Comment:
Only two placeholder test functions in here. Where's the `ReindexTaskCountsHandler` one?

Reply:
Also, the test isn't doing what the convey text says is. Either fix the test or change the Convey. Or at lease add a TODO. I'd probably add a `t.Skip(` too to show they are skipped as not implemented yet.

Reply:
I have done this now. Amazing suggestion by the way